### DISCUSSION TITLE
Custom design token values not being reflected in component

### DISCUSSION THREAD

**Discussion by Scott-McMullan-ABB** [primefaces/community#2773] [PrimeVue]
I have a simple layout that implements a Card.
```vue
<script setup>
import Card from "primevue/card";

const customCard = {
    border: {
        radius: 0
    }
}
</script>

<template>
    <Card :dt="customCard">
        <template #content>
            <slot/>
        </template>
    </Card>
</template>

```

You can see from the component that I want the card to have no border radius, however this is not working. 
<img width="1294" alt="Screenshot 2024-09-30 at 14 59 04" src="https://github.com/user-attachments/assets/d8af0216-411d-4f3a-b997-d75ea0628839">

I've tried adding `root` but this doesn't work either.
```js
const customCard = {
    root: {
        border: {
            radius: 0
        }
    }
}
```

I can acheive the result I want if I update these tokens and save it as a theme preset.
```js
const MyPreset = definePreset(Aura, {
    semantic: {
        ...
    },
    components: {
        ...
        card: {
            border: {
                radius: 0,
            }
        }
    }
})
...
.use(PrimeVue, {
    theme: {
        preset: MyPreset,
        options: {
            // darkModeSelector: 'dark-mode',
        }
    },
    ripple: true
})
```

<img width="1294" alt="Screenshot 2024-09-30 at 15 03 38" src="https://github.com/user-attachments/assets/5bab5cce-6b86-4453-b239-216dddffd8b4">

What am I doing wrong? I don't want every card in my application to have 0 border radius, just the one in this particular component. I cannot seem to get `:dt` to work.

Thanks.

**Comment by j0rgedev** [comment]
Hello @Scott-McMullan-ABB. It seems like there is an open issue for scoped tokens: https://github.com/primefaces/primevue/issues/6502.
I've just reproduced your issue, and it works with vue 3.5.5, but not with 3.5.6

**Reply by Scott-McMullan-ABB** [reply]
That's good to hear, thanks.
